docs/device-agent/register.md

```diff
@@ -39,27 +39,27 @@ _for automatic registration of one or more devices_
 4. Click **Create**
 
 Once the Provisioning Token has been created, you will be shown the 
-**Device Provisioning Credentials** dialog. This is the only time the 
+**Device Provisioning Configuration** dialog. This is the only time the 
 platform will show you this information. 
 Make sure to take a copy or use the **Download** button to save
 the configuration file locally.
 
 ## Connect the device
 
-### Install the credentials
+### Install the configuration
 
 Before you can connect a device to the platform, the device must have
-a **Device Credentials** file or a **Device Provisioning Credentials** 
+a **Device Configuration** file or a **Device Provisioning Configuration** 
 file present in its working directory. There are two ways to do this:
-1. Copy the credentials file into the device's 
+1. Copy the configuration file into the device's 
 [Working Directory](./install.md#working-directory).
-2. Download the credentials file to the device using its built in Web UI.
+2. Download the configuration file to the device using its built in Web UI.
 NOTE: The Device Agent must be running and the command line flag for the Web UI must be enabled.
 See [Command Line Options](./running.md#device-agent-command-line-options) for more information.
 
 ### Copy method
 
-Place the **Device Credentials** or **Device Provisioning Credentials** file on the device
+Place the **Device Configuration** or **Device Provisioning Configuration** file on the device
 in the [Working Directory](./install.md#working-directory)
```

Comment:
```suggestion
Place the **Device Configuration** or **Device Provisioning Configuration** file on the device
in the [Working Directory](./install.md#working-directory)

By default, the agent expects the file to be named `device.yml`, if not, you will need to start the device agent with the `-c` [Command Line Option](./running.md#device-agent-command-line-options) and specify the path of the configuration file.
```

Reply:
I can see anywhere in this doc that mentions the default expected name.

Perhaps a nod to it would help clarify?